recipes-kernel/linux/linux-mainline/debug-fs.cfg

```diff
@@ -0,0 +1,26 @@
+# CONFIG_GENERIC_IRQ_DEBUGFS is not set
```

Comment:
Are you sure we need those `is not set` entries?
If we omit them, are these set to `y`?

Reply:
Those configs are likely generated automatically do we want to then modify them by hand?
It takes time to check whether these options can be safely removed and increases chances of mistakes

Reply:
Done [c566016](https://github.com/3mdeb/meta-rte/pull/73/commits/c5660163e0dd179f93452a4884bcf04490a05f6a)